content/styles/01-atoms/_buttons.css

```diff
@@ -16,18 +12,30 @@ button,
 input[type=submit] {
   display: inline-block;
   padding: 0.8rem 2.5rem;
-  color: var(--greenLink);
+  margin: 10px 0;
+  color: var(--primary-color);
   text-align: center;
-  background-color: #fff;
-  border: 0.125rem solid var(--greenLink);
+  background-color: var(--white);
+  border: 0.125rem solid var(--primary-color);
   border-radius: 0.5rem;
   transition: all .3s ease;
 }
 
 .button:hover,
 button:hover,
 input[type=submit]:hover {
-  color: #fff;
-  background-color: var(--greenLinkHover);
-  border: 0.125rem solid var(--greenLinkHover);
+  color: var(--white);
+  background-color: var(--primary-color);
+}
+
+.reverse {
+  background-color: var(--primary-color);
+  color: var(--white);
```

Comment:
Move this to the previous line for consistency (color then background-color)

Reply:
done !